scripts/verify-template.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const puppeteer = require('puppeteer');
+
+/**
+ * This script is used to check for runtime errors in the starter templates
+ * create-sumup-next-app and create-sumup-react-app.
+ */
+(async () => {
```

Comment:
Is this an IIFE? 👀 never had to actually use one before

Reply:
Yes, it is :)